### DISCUSSION TITLE
Unable to View VOD via Link

### DISCUSSION THREAD

**Discussion by CheSauRie** [ant-media/Ant-Media-Server#6812] [FAQs] [answered]
I successfully logged into Ant Media, but I cannot view VOD files through the link:

http://localhost:5080/LiveApp/streams/{vodId}.mp4

It seems like I might need to configure JWT or adjust some settings to access the VOD files. Could you provide guidance on how to resolve this issue?

**Comment by CheSauRie** [comment]
I encountered an issue while using Ant Media Server. Here’s the problem:

When I call Ant Media APIs (e.g., uploading VOD, managing streams) after logging in, everything works fine.
I can view VOD files locally through a link like: http://localhost:5080/LiveApp/{vodId}.mp4
However, when I deploy the server and try to access the same VOD playback link (e.g., http://(deployed-server-ip):5080/LiveApp/{vodId}.mp4), it does not work.

From my understanding, this might be related to JWT or some access restriction in Ant Media Server. Do I need to configure JWT or modify any settings to make the VOD playback link accessible externally?

**Reply by yashtandon113** [reply, marked as answer]
Hi @CheSauRie 

In general, if you are using the AMS with default settings, then your VOD file should play with the below URL format.

http://localhost:5080/LiveApp/streams/vodid.mp4

For example, I have uploaded the VOD file to the external server and it works fine as expected.

https://test.antmedia.io:5443/24x7test/streams/312567925475117055232546.mp4

Please make sure that their is no security token type enabled; otherwise, you need to generate the token for playback.
